store/src/java/com/zimbra/cs/milter/MilterHandler.java

```diff
@@ -354,6 +355,30 @@ private void SMFIC_Rcpt() throws ServiceException {
             connection.send(new MilterPacket(SMFIR_TEMPFAIL));
             return;
         }
+        try { // ZBUG-2165: prevent forwarding to non-permitted distribution list
+            ZimbraLog.milter.trace("evaluating forwarding to non-permitted distribution list");
+            final Account account = prov.getAccountByName(rcpt);
+            ZimbraLog.milter.trace("account: %s", account.getName());
```

Comment:
On line 360 if account is null then at line 361 getName() method will give exception.
Can you move this line inside if block

Reply:
Done